### DISCUSSION TITLE
Build 3.3.3 preventing table schema updates

### DISCUSSION THREAD

**Discussion by campeauslalom** [TabularEditor/TabularEditor3#610] [🙋♀️TE3 Q&A] [answered]
Hello,

I updated to the latest Tabular Editor build (3.3.3) yesterday afternoon, and ran into an issue this morning where when attempting to update table schemas of both new and existing tables or partitions in a model, none of the columns were recognized - so, in the case of new tables there would be partition that was created with no columns, and updating existing tables removed all columns in said table. This behavior was occurring with all of our models whether connecting directly to the server or opening a .bim and using a workspace DB.

I was still able to successfully refresh the model both via XMLA endpoint from Tabular Editor and in the Power BI service without issue. A colleague had not yet installed the newest version and was still on build 3.3.2 and did not run into this issue, and when I uninstalled 3.3.3 and reverted to 3.3.2 I was also once again able to update tables normally.

For reference, these tables use M partitions connected to a Snowflake data warehouse without any transformations - just direct imports. When reverting to 3.3.2, functionality was restored using the same .bim and XMLA endpoints - the other change was the Tabular Editor build.

**Comment by otykier** [comment, marked as answer]
Created issue #612 based on this.
